app/services/moab_on_storage/validator.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
```

Comment:
add comment:  "distinguish between the moab record in the db and the Moab on local storage (and the zipped moab versions replicated to the cloud)"  ?